src/components/stage/stage.css

```diff
@@ -0,0 +1,13 @@
+.stage {
+    /* 
+        Fixes a few extra pixels of margin/padding, that adds on to the bottom 
+        of the element, which messes up the chrome padding consistency. Seems 
+        canvas element is not block by default?
+        https://teamtreehouse.com/community/why-do-we-need-to-set-the-canvas-display-to-block-when-that-is-the-default-display-setting-anyway
+    */
+    display: block; 
+    border-radius: 0.75rem; 
+
+    /* TODO: This is for overriding the value being set somewhere. Where is it being set? */
+    background-color: transparent; 
```

Comment:
Hmm, I'm not sure, but setting `display` to `block` fixes the white space issue.